tests/test_format_and_parse.py

```diff
@@ -1080,3 +1080,8 @@ def test_190(self):
         expected_sql = "SELECT typeof(sum(a3)) FROM a GROUP BY a1"
         expected_json = {'from': 'a', 'select': {'value': {'typeof': {'sum': 'a3'}}}, 'groupby': {'value': 'a1'}}
         self.verify_formatting(expected_sql, expected_json)
+
+    def test_191(self):
+        expected_sql = "SELECT `user ID` FROM a"
+        expected_json = {'select': {'value': '`user ID`'}, 'from': 'a'}
```

Comment:
We should not see the backticks in the expected json.  Maybe there is a bigger problem in `verify_formatting` that is letting this test pass

Reply:
Should I keep this test? 
The other option would be, as you mentioned, modify escape function to use non-ansi quotes (`) for specific column names.

Reply:
This test must change.  Change `expected_json` to not include the backticks:

    expected_json = {'select': {'value': 'user ID'}, 'from': 'a'}

Reply:
Actually, forget my last message, and remove this test.  The `formatting.py` generates double quoted column names

Reply:
I marked test as skipped, so limits of formatting.py are known.